pulp_2to3_migration/app/plugin/docker/repository.py

```diff
@@ -49,7 +49,8 @@ def migrate_to_pulp3(cls, pulp2distributor, repo_version):
                            False if Distribution is an existing one
         """
         if not repo_version:
-            repo_version = pulp2distributor.pulp2_repository.pulp3_repository_version
+            repo = pulp2distributor.pulp2_repos.filter(not_in_plan=False, is_migrated=True)
+            repo_version = repo.pulp3_repository_version
```

Comment:
Can you add a comment in the plugins that this will go away with the simple-complex plan conversion work?